msu/classes/ordered_map.nut

```diff
@@ -48,6 +49,22 @@
 		return ret;
 	}
 
+	function toTable()
+	{
+		return clone this.Table;
+	}
+
+	function addTable( _table, _overwrite = false )
```

Comment:
I would default to true, since that's the standard behavior when using `<-` for a single key.

It's also more performant in situations where this doesn't come up (no `contains` check)